debian/changelog

```diff
@@ -0,0 +1,11 @@
+bosh-agent (2.398.0-0ubuntu1.18.04.1) bionic; urgency=medium
+
+  * Packaging for Bionic
+
+ -- Miriam España Acebal <miriam.espana@canonical.com>  Wed, 10 Nov 2021 11:36:54 +0100
+
+bosh-agent (2.398.0-0ubuntu1) jammy; urgency=medium
+
+  * Initial release based on dh-make-golang
+
+ -- Christian Ehrhardt <christian.ehrhardt@canonical.com>  Tue, 09 Nov 2021 12:02:18 +0100
```

Comment:
We're not releasing it for Jammy, no? Since we're not, we should probably drop this because this gives an idea that it was upload to some place for Jammy -- which isn't the case. :(